htdocs/compta/facture/list.php

```diff
@@ -2400,6 +2400,9 @@
 				// Pending amount
 				if (!empty($arrayfields['rtp']['checked'])) {
 					print '<td class="right nowraponall amount">';
+					if (!empty($obj->paye)) {
+						$remaintopay = 0;
```

Comment:
If remain to pay is 2 euros and we decide to set invoice to status Paid, why hiding the information of remain to pay is 2 ?
We can keep the information of balance on screen because we must know that we set to pay even if balance was not null.
What do you think of keeping the 2 but striked (barré) so we understand balance was not zero and it is no more exoected to be paid.

Reply:
@eldy same sentence in "/compta/facture/card.php" in v18 line 4099 ?

```
if ($object->paye) {
	$resteapayer = 0;
}
```

Reply:
@eldy the balance to pay may be less or more! when you create a credit for an excess payment the invoice is declared paid, we should not see the negative balance if a credit has been created no? Do you have a solution to manage all the cases?

Reply:
> @eldy the balance to pay may be less or more! when you create a credit for an excess payment the invoice is declared paid, we should not see the negative balance if a credit has been created no? Do you have a solution to manage all the cases?

Yes, a good evolution is to not force to 0 the value but showbthe value striked.
I think having on screen a remain to pay = 2 with the 2 striked, we understand that it is not really 2 but 0.